CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# 0.0.98 - 2021-06-11
+
+0.0.98 should be considered a release candidate to the first alpha release of
+rust-lightning and the broader LDK. It represents several years of work
```

Comment:
I think we use "Rust-Lightning" capitalization elsewhere in the codebase. Fine with either but consistency would be nice.